python/ray/llm/_internal/batch/processor/vllm_engine_proc.py

```diff
@@ -80,6 +80,14 @@ class vLLMEngineProcessorConfig(ProcessorConfig):
         description="Whether the input messages have images.",
     )
 
+    # LoRA configurations.
+    dynamic_lora_loading_path: Optional[str] = Field(
+        default=None,
+        description="The S3 path to the dynamic LoRA adapter. "
```

Comment:
These should really be any remote path, right? What about locally mounted shared path?